src/appshell/view/preferences/livebraillepreferencesmodel.h

```diff
@@ -0,0 +1,243 @@
+        "Russian braille [ru-litbrl.ctb]",
+        "Russian braille with indication of capitals [ru-litbrl-detailed.utb]",
+        "Russian braille for program sources [ru-compbrl.ctb]",
+        "Russian contracted braille [ru-ru-g1.ctb]",
+        "Kinyarwanda braille [rw-rw-g1.utb]",
+        "Sanskrit braille [sa.tbl]",
+        "Yakut braille [sah.utb]",
+        "Sindhi braille [sd.tbl]",
+        "Slovak braille [sk-g1.ctb]",
+        "Slovenian braille [sl.tbl]",
+        "Slovenian computer braille [sl-si-comp8.ctb]",
+        "Sesotho uncontracted braille [sot-za-g1.ctb]",
+        "Sesotho contracted braille [sot-za-g2.ctb]",
+        "Serbian braille [sr.tbl]",
+        "Swedish computer braille (1996 standard) [sv-1996.ctb]",
+        "Swedish computer braille (1989 standard) [sv-1989.ctb]",
+        "Swedish uncontracted braille [sv-g0.utb]",
+        "Swedish partially contracted braille [sv-g1.ctb]",
+        "Swedish contracted braille [sv-g2.ctb]",
+        "Swahili grade 1.2 contracted braille [sw-ke-g1-2.ctb]",
+        "Swahili grade 1.3 contracted braille [sw-ke-g1-3.ctb]",
+        "Swahili grade 1.4 contracted braille [sw-ke-g1-4.ctb]",
+        "Swahili grade 1.5 contracted braille [sw-ke-g1-5.ctb]",
+        "Swahili uncontracted braille [sw-ke-g1.utb]",
+        "Swahili grade 2 contracted braille [sw-ke-g2.ctb]",
+        "Tamil braille [ta-ta-g1.ctb]",
+        "Tamil computer braille [ta.tbl]",
+        "Telugu braille [te.tbl]",
+        "Turkish computer braille [tr.tbl]",
+        "Turkish braille [tr-g2.tbl]",
+        "Setswana uncontracted braille [tsn-za-g1.ctb]",
+        "Setswana contracted braille [tsn-za-g2.ctb]",
+        "Tatar braille [tt.utb]",
+        "Ukrainian braille [uk.utb]",
+        "Ukrainian computer braille [uk-comp.utb]",
+        "Uzbek braille [uz-g1.utb]",
+        "Urdu uncontracted braille [ur-pk-g1.utb]",
+        "Urdu contracted braille [ur-pk-g2.ctb]",
+        "Tshivenda uncontracted braille [ve-za-g1.utb]",
+        "Tshivenda contracted braille [ve-za-g2.ctb]",
+        "Vietnamese computer braille [vi.ctb]",
+        "Vietnamese uncontracted braille [vi-vn-g0.utb]",
+        "Vietnamese partially contracted braille [vi-vn-g1.ctb]",
+        "Vietnamese contracted braille [vi-vn-g2.ctb]",
+        "Southern Vietnamese braille [vi-saigon-g1.ctb]",
+        "isiXhosa uncontracted braille [xh-za-g1.utb]",
+        "isiXhosa contracted braille [xh-za-g2.ctb]",
+        "Chinese current braille without tones, for simplified Chinese characters [zh_CHN.tbl]",
+        "Cantonese braille [zh_HK.tbl]",
+        "Taiwanese bopomofo braille [zh-tw.ctb]",
+        "Chinese common braille, for simplified Chinese characters [zhcn-cbs.ctb]",
+        "Chinese current braille with tones [zhcn-g1.ctb]",
+        "Chinese double-phonic braille [zhcn-g2.ctb]",
+        "isiZulu uncontracted braille [zu-za-g1.utb]",
+        "isiZulu contracted braille [zu-za-g2.ctb]",
+    };
```

Comment:
Better move this to like LiveBrailleConfiguration and get this table in load method